pkg/utils/xslice/float.go

```diff
@@ -0,0 +1,123 @@
+/*
+ *
+ * Copyright 2021 waterdrop authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package xslice
+
+import (
+	"sort"
+)
+
+// ContainFloat64 check if target is in ss
+func ContainFloat64(s []float64, target float64) bool {
+	for _, val := range s {
+		if val == target {
+			return true
+		}
+	}
+	return false
+}
+
+// RemoveFloat64 remove empty target elements from ss
+func RemoveFloat64(s []float64, target float64) []float64 {
+	var ret = make([]float64, 0)
+	for _, val := range s {
+		if val != target {
+			ret = append(ret, val)
```

Comment:
It seems to append the elements in the original slice to the new slice one by one, which will continue to trigger expanding and affect performance; idea: set the capacity when initializing, insert directly before finding the target, and after finding the target, the left index equal the right index reduce one